build.gradle

```diff
@@ -135,6 +135,6 @@ subprojects {
 }
 
 // ======== wrapper ========
-task wrapper(type: Wrapper) {
-	gradleVersion = "4.1"
+wrapper {
```

Comment:
Related: https://docs.gradle.org/4.8/release-notes.html#overwriting-gradle's-built-in-tasks